test/test_blob_to_dict_base64.py

```diff
@@ -0,0 +1,21 @@
+import os
+import capnp
+import base64
+import pytest
+
+this_dir = os.path.dirname(__file__)
+
+
+@pytest.fixture
+def blob_schema():
+    return capnp.load(os.path.join(this_dir, "blob_test.capnp"))
```

Comment:
Did you forget to add blob_test.capnp to the commit?

Reply:
thanks, I've added it now